dashboard/app/models/plc/evaluation_answer.rb

```diff
@@ -5,9 +5,9 @@
 #  id                         :integer          not null, primary key
 #  answer                     :string(255)
 #  plc_evaluation_question_id :integer
+#  plc_task_id                :integer
 #  created_at                 :datetime         not null
 #  updated_at                 :datetime         not null
-#  plc_learning_module_id     :integer
```

Comment:
I'm perplexed as to what's happening here...the currently checked in version has plc_learning_module_id annotated. Not sure why your diff is showing it

https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/models/plc/evaluation_answer.rb